README.md

```diff
@@ -108,14 +83,6 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230446442-c9f6074e-ad00-4674-b69b-3f781aaaae0d.png)

**[OPTIONAL]**

- [ ] It would be nice if you don't leave your key features like these 😨 because they make your readme file look not clean 🤔 Kindly put features here like

* Add Vets
* Add Animals
* Add Visits